frontend/src/components/widgets/UserAvatarLink.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import Avatar from "@material-ui/core/Avatar";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/styles";
+import { getUrl as getUserBoardUrl } from "./LinkToUserBoard";
+
+const useStyles = makeStyles({
+  avatarLink: {
+    textDecoration: "none",
+  },
+  container: {
+    display: "flex",
+    alignItems: "center",
+    gap: 8,
+  },
+});
+
+function getAvatarTextInitials(email) {
+  if (email.endsWith("umuzi.org")) {
```

Comment:
there are other learners who are in the bridge course or in BootCamp, and their emails are not umuzis.